packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -168,7 +169,7 @@ function toStringProcessor(value: any): DisplayValue {
 
 export function getRawDisplayProcessor(): DisplayProcessor {
   return (value: any) => ({
-    text: `${value}`,
+    text: getFieldTypeFromValue(value) === 'other' ? `${JSON.stringify(value)}` : `${value}`,
```

Comment:
This processor will be used in more scenarios so I'm a bit hesitant regarding this change. Would love to get your input here @torkelo @ryantxu.

Reply:
Also - what if `JSON.stringify` throws an error (e.g. with an object with circular structure)?

Reply:
@ifrost good catch, if we are fine with the above changes means, I can update it a replacerFunc to the `JSON.stringify` to fix the circular object issue.